localstack/services/awslambda/lambda_api.py

```diff
@@ -1078,12 +1078,15 @@ def generic_handler(*_):
     if not lambda_cwd:
         return
 
-    # get local lambda working directory
+    # get local lambda code archive path
     tmp_file = os.path.join(lambda_cwd, LAMBDA_ZIP_FILE_NAME)
 
     if not zip_file_content:
```

Comment:
```suggestion
    if not zip_file_content:
         zip_file_content = load_file(tmp_file, mode="rb")
    else:
        # override lambda archive with fresh code if we got an update
        save_file(tmp_file, zip_file_content)
```
Tiny performance optimization, to avoid load->store roundtrip if not required.